### DISCUSSION TITLE
Generated TS types being split into seperate folders within node modules.

### DISCUSSION THREAD

**Discussion by taylorcantwell** [prisma/prisma#10466] [Q&A] [answered]
Has anyone got any idea why my generated TS types are being split between “.prisma/client” and “@prisma/client”?  For whatever reason the model types are being placed in “@prisma/client” whereas the input types are being placed in “.prisma/client”? The docs demo them being both pulled from “@prisma/client”.

**Comment by TasinIshmam** [comment, marked as answer]
Hey @taylorcantwell 

Looking at the generated  `@prisma/client` I see that it simply imports all the types and models from `.prisma/client`. So technically you could import types/models/inputs etc from either location. 

If this is behavior is different for you? If so, could you provide a way to reproduce it?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
